### DISCUSSION TITLE
Create Custom Manifest

### DISCUSSION THREAD

**Discussion by fguisso** [goreleaser/goreleaser#3518] [Q&A] [answered]
Hey ppl!

I'd need to create a `manifest.json` with particular fields, I don't find anything about that, just vendor manifests.

If nothing exists, an excellent feature to add is to accept Custom Manifest. With this, people can use our unique schemas.

This is why I need a Custom Manifest:
- [create a new manifest file](https://github.com/tsuru/rpaas-operator/issues/124)
- [usage of manifest file](https://github.com/tsuru/tsuru-client/issues/172)

**Comment by caarlos0** [comment, marked as answer]
Sorry, I don't understand which manifest you're talking about... 

You want to create a new "custom" json from your release artifacts, is that it?

If so, it's not possible from goreleaser natively.
You could however write something else and run it after goreleaser reading the dist/artifacts.json file...
